spec/quality_spec.rb

```diff
@@ -228,6 +228,10 @@ def check_for_specific_pronouns(filename)
 
         # there's no way around this warning
         last_command.stderr.sub!(/^YAML safe loading.*/, "")
+        last_command.stderr.sub!("WARNING:  No cert chain found. Please consider signing your gem to improve security.", "")
+        last_command.stderr.sub!("See https://guides.rubygems.org/security for more information.", "")
+        last_command.stderr.sub!("WARNING:  See http://guides.rubygems.org/specification-reference/ for help", "")
+        last_command.stderr.sub!("\n\n", "")
```

Comment:
Nitpick / curiosity, why do we need to remove two new line characters if we're removing three warning lines? I would instead include the line character with the previous `gsub`'s.